web/js/containers/animation-widget.js

```diff
@@ -457,10 +459,13 @@ class AnimationWidget extends React.Component {
       hasSubdailyLayers,
     } = this.props;
     const { speed, widgetPosition } = this.state;
+    const cancelSelector = '.no-drag, .interval-btn-container, .date-arrows';
```

Comment:
You may be able to remove `.interval-btn-container,` from this string since `.no-drag` is added to that element already.

Reply:
Nice, good catch!